src/SelectField/SelectField.jsx

```diff
@@ -20,9 +20,15 @@ function getStyles(props) {
 }
 
 const SelectField = React.createClass({
+  displayName: 'SelectField',
 
   propTypes: {
     /**
+     * opens DropDownMenu and triggers focus of TextField
+     */
+    autoFocus: React.propTypes.bool,
```

Comment:
No, you called it `autoFocus`, which is different to  `openImmediately` in `DropDownMenu`.  My point was, lets be consistent with naming, even if the naming isn't great (unless you feel there's a good reason for it to be different)...